packages/experimental/MenuButton/src/renderContextualMenu.tsx

```diff
@@ -0,0 +1,48 @@
+/* @jsxFrag React.Fragment */
+import React, { useState, useCallback } from 'react';
+import { ContextualMenu, ContextualMenuItem, SubmenuItem, Submenu, ContextualMenuProps } from '@fluentui-react-native/contextual-menu';
+import { MenuButtonItemProps } from './MenuButton.types'
+
+export const renderContextualMenu = (contextualMenu: ContextualMenuProps, menuItems: MenuButtonItemProps[]): JSX.Element => {
+    const menuItemsUpdated = menuItems?.map((item) => {
+      if (item.hasSubmenu) {
+        const [showSubmenu, setShowSubmenu] = useState(false);
+        const toggleShowSubmenu = useCallback(() => {
+          setShowSubmenu(!showSubmenu);
+        }, [showSubmenu, setShowSubmenu]);
+        const onDismissSubmenu = useCallback(() => {
+          setShowSubmenu(false);
+        }, [setShowSubmenu]);
+        const { onHoverIn = toggleShowSubmenu, submenuProps = {}, ...restItems } = item;
+        const { onDismiss = onDismissSubmenu, setShowMenu = toggleShowSubmenu, ...restSubmenuProps } = submenuProps;
+        const menuItemUpdated = {
+          ...restItems,
+          onHoverIn,
+          showSubmenu: item.showSubmenu ?? showSubmenu,
+          submenuProps: { ...restSubmenuProps, onDismiss, setShowMenu },
+        };
+        return menuItemUpdated;
+      }
+      return item;
+    });
+
+    return (<ContextualMenu {...contextualMenu}>
+          {menuItemsUpdated?.map((menuItem) => {
```

Comment:
Indentation seems off. If you run `yarn prettier-fix` on this package do you still keep the indentation?

Reply:
I have it in my VSCode installed. I ran prettier-fix but it failed with one of the components

Reply:
Prettier should work now. I fixed it on Friday.